examples/webxr_vr_teleport.html

```diff
@@ -0,0 +1,278 @@
+		</div>
+
+		<!-- Import maps polyfill -->
+		<!-- Remove this when import maps will be widely supported -->
+		<script async src="https://unpkg.com/es-module-shims@1.3.6/dist/es-module-shims.js"></script>
+
+		<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.module.js"
+				}
+			}
+		</script>
+
+		<script type="module">
+
+			import * as THREE from '../build/three.module.js';
+
+			import { BoxLineGeometry } from './jsm/geometries/BoxLineGeometry.js';
+			import { VRButton } from './jsm/webxr/VRButton.js';
+			import { XRControllerModelFactory } from './jsm/webxr/XRControllerModelFactory.js';
+
+			let camera, scene, raycaster, renderer;
+			let controller1, controller2;
+			let controllerGrip1, controllerGrip2;
+
+			let room, marker, floor;
+
+			let INTERSECTED;
+			const tempMatrix = new THREE.Matrix4();
+
+			const clock = new THREE.Clock();
+
+			init();
+			animate();
+
+			function init() {
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 0x505050 );
+
+				camera = new THREE.PerspectiveCamera( 50, window.innerWidth / window.innerHeight, 0.1, 10 );
+				camera.position.set( 0, 1, 3 );
+
+				room = new THREE.LineSegments(
+					new BoxLineGeometry( 6, 6, 6, 10, 10, 10 ).translate( 0, 3, 0 ),
+					new THREE.LineBasicMaterial( { color: 0x808080 } )
+				);
+				scene.add( room );
+
+				scene.add( new THREE.HemisphereLight( 0x606060, 0x404040 ) );
+
+				const light = new THREE.DirectionalLight( 0xffffff );
+				light.position.set( 1, 1, 1 ).normalize();
+				scene.add( light );
+
+				marker = new THREE.Mesh(
+					new THREE.CircleGeometry( 0.25, 32 ).rotateX( - Math.PI / 2 ),
+					new THREE.MeshBasicMaterial( { color: 0x808080 } )
+				);
+				scene.add( marker );
+
+				floor = new THREE.Mesh(
+					new THREE.PlaneGeometry( 4.8, 4.8, 2, 2 ).rotateX( - Math.PI / 2 ),
+					new THREE.MeshBasicMaterial( { color: 0x808080, transparent: true, opacity: 0.25 } )
+				);
+				scene.add( floor );
+
+				raycaster = new THREE.Raycaster();
+
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.outputEncoding = THREE.sRGBEncoding;
+				renderer.xr.enabled = true;
+				document.body.appendChild( renderer.domElement );
+
+				//
+
+				document.body.appendChild( VRButton.createButton( renderer ) );
+
+				// controllers
+
+				function onSelectStart() {
+
+					this.userData.isSelecting = true;
+
+				}
+
+				function onSelectEnd() {
+
+					this.userData.isSelecting = false;
+
+					if ( INTERSECTED ) {
+
+						const baseReferenceSpace = renderer.xr.getReferenceSpace();
+						const offsetPosition = { x: - INTERSECTED.x, y: - INTERSECTED.y, z: - INTERSECTED.z, w: 1 };
+						const offsetRotation = new THREE.Quaternion();
+						const transform = new XRRigidTransform(offsetPosition, offsetRotation);
```

Comment:
Here and in the next line the spaces between parameters and brackets are missing.

Reply:
Nice catch. Fixed.